cli/cmd/profile.go

```diff
@@ -98,14 +101,30 @@ func newCmdProfile() *cobra.Command {
 				return err
 			}
 
+			var k kubernetes.Interface
+			k, err = k8s.NewAPI(kubeconfigPath, kubeContext, 0)
```

Comment:
any reason not to just do `k, err :=` here instead of declaring k on the previous line?